<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>botgram</class>
 <widget class="QMainWindow" name="botgram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1310</width>
    <height>810</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1310</width>
    <height>810</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1310</width>
    <height>810</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>botgram</string>
  </property>
  <property name="styleSheet">
   <string notr="true">btn_verify:hover
{

	background-color: rgb(31, 162, 190);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1310</width>
      <height>810</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1310</width>
        <height>810</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1310</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1310</width>
        <height>810</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="DataFiles/files.qrc">:/new/prefix1/pishvazThemBotGram.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="StartApplication_btn">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>530</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Go to Sign in Form</string>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;
&lt;style&gt;
StartApplication_btn : hover
{
border: 10px black solid;

}
&lt;/style&gt;
&lt;/html&gt;
</string>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: #E6C323;
color:#ffffff;
border-radius:25px;

</string>
      </property>
      <property name="text">
       <string>Let's Go ...</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="tozihat_lbl">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>340</y>
        <width>431</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Here you can send your message resive&lt;/p&gt;&lt;p&gt; youre photo,video and youre vioces.&lt;/p&gt;&lt;p&gt; Also you can choose youre them.&lt;/p&gt;&lt;p&gt;We propare to you that can build and use from Bots&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="wellcom_lbl">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wellcome to</string>
      </property>
     </widget>
     <widget class="QLabel" name="Botgram_lbl">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>531</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>50</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>BotGram</string>
      </property>
     </widget>
     <widget class="QLabel" name="LearnMore_lbl">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>540</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://www.google.com&quot;&gt;&lt;span style=&quot; font-size:14pt; text-decoration: underline; color:#E6C323;&quot;&gt;google&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color : #B5E61D;</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://www.roshd.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#E6C323;&quot; &gt;Learn more &amp;gt;&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard</set>
      </property>
      <property name="buddy">
       <cstring></cstring>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="styleSheet">
      <string notr="true">btn_verify
{
background-color: rgb(96, 204, 243,0);
color: rgb(131, 155, 166);
border-radius:25px;
border:1px solid;
border-color: rgb(131, 155, 151);
}</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1310</width>
        <height>810</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="DataFiles/files.qrc">:/new/prefix1/LoginAndSignIn.jpg</pixmap>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <widget class="QFrame" name="wi_login">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>0</y>
        <width>601</width>
        <height>691</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">btn_verify:hover
{

	background-color: rgb(35, 154, 186);
}</string>
      </property>
      <widget class="QLineEdit" name="txt_username">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>170</y>
         <width>381</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 20px;
    background-color:#00000000 ;

 border: 1px solid;
border-color: rgb(131, 155, 151);

padding-left:10px;
color:rgb(255, 255, 255);</string>
       </property>
       <property name="placeholderText">
        <string>User name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_password">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>310</y>
         <width>381</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 20px;
    background-color:#00000000 ;

 border: 1px solid;
border-color: rgb(131, 155, 151);
padding-left:10px;</string>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_email">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>450</y>
         <width>381</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 20px;
    background-color:#00000000 ;
 border: 1px solid;
border-color: rgb(131, 155, 151);
padding-left:10px;</string>
       </property>
       <property name="placeholderText">
        <string>E-mail</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_verify">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>620</y>
         <width>381</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">

color: rgb(131, 155, 166);
border-radius:25px;
border:1px solid;
border-color: rgb(131, 155, 151);

</string>
       </property>
       <property name="text">
        <string>Verify</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_alarm">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>250</y>
         <width>381</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>381</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="hlayout_alarm" stretch="1,20">
         <item>
          <widget class="QLabel" name="lbl_pic_alarm">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="font">
            <font>
             <stylestrategy>PreferDefault</stylestrategy>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:red;
border-radius:5px;
</string>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_Alarm">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_alarm_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>390</y>
         <width>381</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>381</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="hlayout_alarm_2" stretch="1,20">
         <item>
          <widget class="QLabel" name="lbl_pic_alarm2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="font">
            <font>
             <stylestrategy>PreferDefault</stylestrategy>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:red;
border-radius:5px;
width:20px;
height:20px;</string>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_alarm2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <zorder>txt_email</zorder>
      <zorder>txt_username</zorder>
      <zorder>txt_password</zorder>
      <zorder>btn_verify</zorder>
      <zorder>frame_alarm</zorder>
      <zorder>frame_alarm_2</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{
	background-color: rgb(65, 205, 82);
}
QLineEdit
{
border:1px		solid		gray;
border-radius:8px;
font-size:14px;
min-height:25px;
}

QLineEdit:focus
{
border:1px		solid		#03dac6;
border-radius:4px;

}
*{
color:black;
}
</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="DataFiles/files.qrc"/>
 </resources>
 <connections/>
</ui>
